omod/src/main/java/org/openmrs/module/rwandahivflowsheet/web/model/FormDataModel.java

```diff
@@ -432,21 +434,33 @@ public String getCurrentDrugNames(String drugSetName) {
 			if (o.getDrug() != null)
 				ret.append(o.getDrug().getName());
 			else
-				ret.append(o.getConcept().getBestName(Context.getLocale()).getName());
+				ret.append(o.getConcept().getName(Context.getLocale()).getName());
 			if (i.hasNext())
 				ret.append(" ");
 		}
 		return ret.toString();
 	}
 	
-	
+	public List<DrugOrder> getDrugOrdersByPatient(Patient patient){
+		List<DrugOrder> ret = new ArrayList<DrugOrder>();
+		List<Order> allPatientOrders = Context.getOrderService().getAllOrdersByPatient(patient);
```

Comment:
How about using something like this? https://pastebin.com/x9PWZ3Q8

Reply:
Thanks a lot! [Here](https://github.com/PIH/openmrs-module-rwandahivflowsheet/pull/2/commits/49ca7831c7a2659a22130e2b8953c95addc2270b) is the commit.